scrapers/wv/events.py

```diff
@@ -101,6 +97,12 @@ def scrape_meeting_page(self, url):
         when = when.split("-")[0]
         when = self.clean_date(when)
         when.replace("Thursady", "Thursday")
+
+        # Remove all text after the third comma to make sure no extra text
+        # is included in the date. Required to correctly parse texxt like this:
+        # "Friday, March 3, 2023, Following wrap up of morning agenda"
+        when = ",".join(when.split(",")[:2])
```

Comment:
Good idea; can you move this down to the clean_date function where the rest of the date logic lives?

Reply:
Fixed with https://github.com/openstates/openstates-scrapers/pull/4521/commits/34d9c100bd25867cc4956bcfcbe46dc9eefcf7fa